pkg/app/pipectl/cmd/application/add.go

```diff
@@ -59,7 +59,7 @@ func newAddCommand(root *command) *cobra.Command {
 
 	cmd.Flags().StringVar(&c.repoID, "repo-id", c.repoID, "The repository ID. One the registered repositories in the piped configuration.")
 	cmd.Flags().StringVar(&c.appDir, "app-dir", c.appDir, "The relative path from the root of repository to the application directory.")
-	cmd.Flags().StringVar(&c.configFileName, "config-file-name", c.configFileName, "The configuration file name. Default is .pipe.yaml")
+	cmd.Flags().StringVar(&c.configFileName, "config-file-name", c.configFileName, "The configuration file name")
```

Comment:
sorry, the default value for this field is no longer allowed?

Reply:
No, this change removes just the message. This flag package automatically emits the default value like:
```
The configuration file name. (default "app.pipecd.yaml")
```

Reply:
Currently it's duplicated: 
```
The configuration file name. Default is .pipe.yaml (default ".pipe.yaml")
```

Reply:
ah got it, thanks 👍